client/signup/config/flows.js

```diff
@@ -378,10 +381,8 @@ const Flows = {
 			flow = removeUserStepFromFlow( flow );
 		}
 
-		// Show design type with store option only to new users with EN locale.
-		if ( ! user.get() && 'en' === i18n.getLocaleSlug() ) {
-			flow = filterDesignTypeInFlow( flow );
-		}
+		// Maybe modify the design type step to a variant with store
+		flow = filterDesignTypeInFlow( flow );
```

Comment:
If we turn `signup/atomic-store-flow` off with the feature flag, will the `design-type-with-store` be enabled for all users, not just new ones and EN locale? If yes, I don't think that's wanted. Could we maybe move the feature flag check outside of the `filterDesignTypeInFlow` function?

Reply:
Yes, it will be enabled for all users. Do we want to keep the old behavior intact when the feature flag is disabled, and use the new behavior (show store to all, not only new+EN) only when the flag is enabled?

That makes sense, I'll fix that.

Reply:
Yeah, I think we want to retain the current behavior if the flag is turned off. Just my guess.